### PR TITLE
Improving dev cross-platform support

### PR DESCRIPTION
**Pull request by TomBillingtonUK**
This PR:
- Fixes the lint:scss npm command so it runs cross-platform 
- Adds windows specific logic to the lint gulp tasks
- Introduces the npm package cross env to allow tests to work cross-platform

Changes have been tested on both a windows 11 machine and a MacBook running Monterey

### CODE REVIEW

**Comment by TomBillingtonUK** [comment]
Linked to issue #2571

**Review by vanitabarrett — APPROVED** [review]
Thanks for raising this PR @TomBillingtonUK 

These changes look good to me. I've tested on Windows and can 1) reproduce the behaviour before this change, and 2) confirm that these changes allow the commands to be run successfully on Windows.

This will need a second review from another developer in the team before it can be merged.

**Review by domoscargin — APPROVED** [review]
Can replicate and fix on my copy of Windows 10.

**Review by querkmachine — APPROVED** [review]
Have also just reproduced the issue and the fix on Windows 11 😁

**Pull request merged**
